src/test/resources/fastapi/main.py

```diff
@@ -18,22 +17,21 @@ class Item(BaseModel):
     validUntil: date = None
     revisedAt: list[date] = []
 
+class ItemFilter(BaseModel):
+    item_ids: list[str] = []
 
 items = {
-    1: Item(name="Portal Gun", price=42.0, tags=["sci-fi"], revisedAt=["2007-10-10", "2022-12-08"]),
-    2: Item(name="Plumbus", price=32.0, validUntil="2999-01-01", properties={"feeble": "schleem"}),
+    1: Item(id="1", name="Portal Gun", price=42.0, tags=["sci-fi"], revisedAt=["2007-10-10", "2022-12-08"]),
+    2: Item(id="2", name="Plumbus", price=32.0, validUntil="2999-01-01", properties={"feeble": "schleem"}),
 }
 
-
 @app.get("/")
 def root():
     return {"Hello": "World"}
 
-
-@app.get("/search")
-def search_items(q: Union[str, None] = None) -> list[Item]:
-    return items.values()
-
+@app.post("/search")
```

Comment:
Wouldn't switching to POST affect any existing tests?

Reply:
Apparently it was not used...